extensions/amp-video-docking/0.1/amp-video-docking.js

```diff
@@ -613,9 +616,13 @@ export class VideoDocking {
         this.dock_(video, target, /* step */ 1);
         return;
       }
-      if (this.isCurrentlyDocked_(video)) {
+      // no undocking when height changed (e.g. some browsers dynamically show/hide
+      // the URL bar during scroll, which leads to a viewport resize)
+      const widthChanged = this.lastWidth_ !== this.viewportRect_.width;
```

Comment:
@alanorozco thanks for your feedback. It makes sense to store the full viewportRect instead of the individual width value and I've applied your proposed changes.